Gemfile.lock

```diff
@@ -198,23 +193,23 @@ GEM
     i18n (0.9.5)
       concurrent-ruby (~> 1.0)
     ice_nine (0.11.2)
-    image_processing (1.9.3)
+    image_processing (1.10.0)
```

Comment:
### [image_processing](https://github.com/janko/image_processing)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.9.3 → 1.10.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/janko/image_processing/compare/v1.9.3...v1.10.0).

These are the individual commits:
- (9fb6a0b) [Add table of contents](https://github.com/janko/image_processing/commit/9fb6a0b9eee17488c3cd06b0095776000bfe043b)
- (979ec78) [Remove -regard-warnings from examples](https://github.com/janko/image_processing/commit/979ec7826d8ba3b282ea8e80309095fab8d504a3)
- (8b48f2e) [[minimagick] Add :loader for explicitly setting input file t…](https://github.com/janko/image_processing/commit/8b48f2e9198bde34789e19c35a293b44817d1258)
- (d035c16) [Bump to 1.10.0](https://github.com/janko/image_processing/commit/d035c16bf16df41eda69c86bebaafc03e04dc53c)


</details>